database/src/main/resources/migrations/ddl/V1.0.5__users_add_verification_columns.sql

```diff
@@ -0,0 +1,39 @@
+-- Step 1: Remove the 'verified' column
+ALTER TABLE users
+    DROP COLUMN verified;
+
+-- Step 2: Modify 'verification_token' to allow NULL values
+ALTER TABLE users
+    ALTER COLUMN verification_token DROP NOT NULL;
+
+-- Step 3: Add new columns 'created_at', 'last_sent_verification_notification', and 'verified_at'
+ALTER TABLE users
+    ADD COLUMN created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP;
+
+ALTER TABLE users
+    ADD COLUMN last_sent_verification_notification TIMESTAMP;
+
+ALTER TABLE users
+    ADD COLUMN verified_at TIMESTAMP;
+
+ALTER TABLE users
+    ADD COLUMN last_changed_password TIMESTAMP;
+
+ALTER TABLE users
```

Comment:
(Optional)
The user table starts to grow pretty large.
We could extract the tokens in a new table

The token table would contain
token VARCHAR
type VARCHAR (VERIFICATION / PASSWORD_RESET enum)
used BOOLEAN
created_at TIMESTAMP
expires_at TIMESTAMP